packages/babel-helper-create-class-features-plugin/src/decorators.ts

```diff
@@ -40,40 +40,57 @@ function takeDecorators(node: Decorable) {
   return result;
 }
 
-function getKey(node) {
-  if (node.computed) {
+type AcceptedElement = Exclude<ClassElement, t.TSIndexSignature>;
+type SupportedElement = Exclude<
+  AcceptedElement,
+  t.ClassPrivateMethod | t.ClassPrivateProperty | t.ClassAccessorProperty
+>;
+
+function getKey(node: SupportedElement) {
+  if (node.type === "StaticBlock") {
```

Comment:
Maybe we should throw plugin ordering errors for `StaticBlock` seen here.

Reply:
This seems to be broken regardless of the plugin ordering, when using this test:
```js
const code = `
@dec
class A {
  static {
    x = 2;
  }
}
`;

const out = babel.transformSync(code, {
  configFile: false,
  plugins: [
    [
      require("@babel/plugin-proposal-decorators"),
      { version: "2018-09", decoratorsBeforeExport: true },
    ],
    require("@babel/plugin-proposal-class-static-block"),
  ],
});
```

in `main` it throws; with this PR it discards the static block. Lets keep this PR as internal-only, but that bug should then be fixed.

Reply:
```suggestion
  if (node.type === "StaticBlock") {
    // TODO: Fix compatibility between 2018-09 decorators and static blocks
```

Reply:
Ok actually, lets just add an error as you did for accessor properties. If someone wants to use static blocks they can upgrade their code to use the new decorators proposal.